code/game/objects/items/plushes.dm

```diff
@@ -542,3 +542,133 @@
 	attack_verb = list("stung")
 	gender = FEMALE
 	squeak_override = list('sound/voice/moth/scream_moth.ogg'=1)
+
+/obj/item/toy/plush/goatplushie
+	name = "strange goat plushie"
+	icon_state = "goat"
+	desc = "Despite its cuddly appearance and plush nature, it will beat you up all the same. Goats never change."
+
+/obj/item/toy/plush/goatplushie/angry
+	var/mob/living/carbon/target
+	throwforce = 6
+	var/cooldown = 0
+	var/cooldown_modifier = 20
+
+/obj/item/toy/plush/goatplushie/angry/Initialize()
+	. = ..()
+	START_PROCESSING(SSprocessing, src)
+
+/obj/item/toy/plush/goatplushie/angry/process()
+	if (prob(25) && !target)
+		var/list/targets_to_pick_from = list()
+		for(var/mob/living/carbon/C in view(7, src))
+			if(considered_alive(C.mind))
+				targets_to_pick_from += C
+		if (!targets_to_pick_from.len)
+			return
+		target = pick(targets_to_pick_from)
+		visible_message("<span class='notice'>[src] stares at [target].</span>")
+	if (world.time > cooldown && target)
+		ram()
+
+/obj/item/toy/plush/goatplushie/angry/proc/ram()
+	if(prob((obj_flags & EMAGGED) ? 98:90) && isturf(loc) && considered_alive(target.mind))
+		throw_at(target, 10, 10)
+		visible_message("<span class='danger'>[src] rams [target]!</span>")
+		cooldown = world.time + cooldown_modifier
+	target = null
+	visible_message("<span class='notice'>[src] looks disinterested.</span>")
+
+/obj/item/toy/plush/goatplushie/angry/emag_act(mob/user)
+	if (obj_flags&EMAGGED)
+		visible_message("<span class='notice'>[src] already looks angry enough, you shouldn't anger it more.</span>")
+		return
+	cooldown_modifier = 5
+	throwforce = 20
```

Comment:
That'd be a pretty lethal 20 force with a 0.5 second cooldown, if not for the fact that it has to wait for processing anyway, making the cooldown reduction more or less irrelevant since it's 2 seconds anyway.

Just to be sure, was the intention to make it 20 and 5 seconds instead?

Reply:
don't forget that the goat plushie has to win a 25% chance roll every time it wants to attack (so that it can actually find a target), since it discards its saved target after every attack)

Reply:
That's why i'm asking; that fact makes all variations of cooldown_modifier in the PR completely irrelevant